build-tools/src/main/resources/hawkular-checkstyle/checkstyle.xml

```diff
@@ -54,6 +54,13 @@
       <property name="processJavadoc" value="true" />
     </module>
 
+    <module name="ImportOrder">
+        <property name="groups" value="/^javax?\./,org,com"/>
```

Comment:
Yes, it does. E.g. for `org.hawkular.component.availcreator.AvailPublisher`, it enforces the order we already have in place https://github.com/hawkular/hawkular/blob/f2031fc1bfd602f708cbe77cde4a6c9102c233ef/modules/avail-creator/src/main/java/org/hawkular/component/availcreator/AvailPublisher.java#L19-L25